gce/notebook/Dockerfile

```diff
@@ -25,26 +25,28 @@ RUN conda install --yes -c conda-forge \
     python-blosc \
     scipy \
     scikit-image \
+    tornado=4.5 \
     xarray \
     zict \
     && conda clean -tipsy
 
-RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager @jupyterlab/hub-extension
-
 RUN pip install fusepy click jedi kubernetes --upgrade --no-cache-dir
 
 RUN pip install daskernetes==0.1.3 \
+                distributed==1.21.3 \
```

Comment:
do we need this and the conda version of 1.21.2?

Reply:
Currently defaults doesn't have 1.21.3, so I'm adding on an extra pip install for now.

Reply:
I've now removed distributed from the conda line above